content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,191 @@
+- [French](https://linuxfr.org/tags/xmpp/public)
+- [Spanish](https://www.jabber.de/)
+
+## Software releases
+
+It has been almost a year since the release of Gajim 1.1.3. A year of developing new features, cleaning up old code, and fixing bugs. This month, it is finally the time for a [first beta release of Gajim 1.2](https://gajim.org/post/2020-05-08-gajim-1.2.0-beta1-released/). Highlights are (amongst others): improved group chat system, completely rewritten network code, and a new account creation assistant. But there is much more to discover.
+
+Log in:
+![Gajim 1.2.0-beta1 log in](https://gajim.org/img/news/account-creation-1.png)
+
+Sign up:
+![Gajim 1.2.0-beta1 sign up](https://gajim.org/img/news/account-creation-2.png)
+
+Create account:
+![Gajim 1.2.0-beta1 create account](https://gajim.org/img/news/account-creation-3.png)
+
+### Clients and applications
+
+[Gajim Development News May 2020](https://gajim.org/post/2020-05-26-development-news-may/): New account badges and the great plugin overhaul. In preparation of the upcoming 1.2 release, many plugins have been revisited. Check out the new file preview!
+
+Goffi released [Salut à Toi progress note for calendar week 21](https://www.goffi.org/b/kmebCba3m38dr9sgzgGrkp/progress-note)!
+
+[Poezio](https://poez.io) released 0.13.0 and 0.13.1 which now requires python 3.7. These versions bring Poezio into the club of OMEMO supporting clients and also enable HTTP upload via dedicated plugins. Check the full [changelog](https://lab.louiz.org/poezio/poezio/-/blob/master/CHANGELOG) to look at all the improvements of these versions.
+
+[Biboumi](https://biboumi.louiz.org/), the XMPP/IRC gateway, was released in version 0.8.5, fixing compilation issues with GCC 10.
+
+The python based XMPP automatic file sender [Errol has been updated to v2.0.1](https://blog.agayon.be/errol_update1.html) which modernizes its async I/O calls and adds more privacy. By moving from 'aionotify' to the better maintained 'watchdog', Errol theoretically extends its support (from Linux only) to Windows, Mac OS X and FreeBSD.
+
+## Services
+
+blabber.im now [supports IPv6](https://blabber.im/blabber-im-ist-nun-ipv6-tauglich/). The service operators created an [invitation link which works for their server but also for every other ID](https://blabber.im/en/i-moved/). If you moved to an XMPP based chat client, you can now conveniently put this link into the status of your previous non-XMPP chat clients. This link offers your contacts all the necessary infos to be able to reach you and (if they want to) follow you into the XMPP world.
+
+![Jabber.org](https://www.jabber.org/jabber.png)
+
+jabber.org migrates its server to Prosody. [Stay informed!](https://www.jabber.org/notices.html)
+
+yax.im rolled out [new anti-spam measures](https://yaxim.org/blog/2020/05/12/new-anti-spam-measures/). You may be interested in the '[The Jabber Spam Fighting Manifesto](https://github.com/JabberSPAM/jabber-spam-fighting-manifesto)', too.
```

Comment:
Warning: `yax.im` renders on Tinyletter, but not on GitHub not Atom (by GitHub), it might be the case on Pelican.